discordrb.json

```diff
@@ -57,6 +57,28 @@
             "voiceReceive": "no",
             "voiceSend": "yes",
             "voiceStateUpdate": "yes"
+        },
+        "misc": {
+            "userLoginWithPassword": "yes",
+            "userLoginWithToken": "yes",
+            "botLogin": "yes",
+            "cachesTokens": "yes",
+            "apiDocs": "automatic",
+            "apiDocsUrl": "http://www.rubydoc.info/gems/discordrb",
```

Comment:
URL to the API docs. This will have to be shown together with `apiDocs` as a HTML link.